### PR TITLE
Magento 2.4.* fix

### PR DESCRIPTION
**Pull request by kylianleemkuil**
+ Renamed some keys for Magento 2.4.* 
-  Deleted some Constraints for Magento 2.4.* 
+ Added checks for doubles in tables
-  Delete row_id column again

### CODE REVIEW

File: scripts/product.sql [hunk 1]
```diff
@@ -287,81 +287,88 @@ ALTER TABLE `catalog_product_bundle_selection`
...
     DROP FOREIGN KEY `CAT_PRD_ENTT_TIER_PRICE_ROW_ID_CAT_PRD_ENTT_ROW_ID`,
-    DROP INDEX `UNQ_E8AB433B9ACB00343ABB312AD2FAB087`,
+    DROP INDEX `UNQ_EBC6A54F44DFA66FA9024CAD97FED6C7`,
     ADD CONSTRAINT `UNQ_E8AB433B9ACB00343ABB312AD2FAB087` UNIQUE KEY (`entity_id`,`all_groups`,`customer_group_id`,`qty`,`website_id`),
-    DROP COLUMN `row_id`;
+	DROP COLUMN `row_id`;
+
+--wishlist 
+ALTER TABLE `wishlist`
+	DROP INDEX `WISHLIST_CUSTOMER_ID`;
```
Comment (thomas-kl1) [inline scripts/product.sql:368]: Why this index needs to be removed? Everything else looks good to me :)

Comment (kylianleemkuil) [reply scripts/product.sql:368]: When I executed php bin/magento setup:upgrade it gave an error on that index. When I deleted that index it suddenly finished the setup:upgrade and everything was working fine.

Comment (thomas-kl1) [reply scripts/product.sql:368]: Do you know what was the error? Actually this index is added by the wishlist module shipped with the Open-Source version

Comment (kylianleemkuil) [reply scripts/product.sql:368]: It said it had a double index WISHLIST_CUSTOMER_ID so it needed to be removed before setup:upgrade could finish.

Comment (thomas-kl1) [reply scripts/product.sql:368]: ok, I guess it's related to declarative schema then, could you add a little comment above this line with this short explanation?

Comment (kylianleemkuil) [reply scripts/product.sql:368]: Just checked again in a my test database of a enterprise version and the wishlist table is not even there. So it is apparently not possible to do before migrating and needs to be manually. I will delete the comment and make a new pull request.

**Review by thomas-kl1 — APPROVED** [review]

**Comment by thomas-kl1** [comment]
@kylianleemkuil  Thanks for the contribution and fixes!!! :partying_face:

**Pull request merged**